ush/forecast_postdet.sh

```diff
@@ -904,7 +913,15 @@ CICE_postdet() {
 
   # Copy CICE ICs
   echo "Link CICE ICs"
-  cice_restart_file="${COM_ICE_RESTART_PREV}/${PDY}.${cyc}0000.cice_model.res.nc"
+  if [[ "${DOIAU}" = "YES" ]]; then
+    sCDATE=$(date -d "${current_cycle:0:8} ${current_cycle:8:2} - 3 hours" +%Y%m%d%H)
+    sPDY="${sCDATE:0:8}"
+    scyc="${sCDATE:8:2}"
+    cice_restart_file="${COM_ICE_RESTART_PREV}/${sPDY}.${scyc}0000.cice_model.res.nc"
+  else
+    cice_restart_file="${COM_ICE_RESTART_PREV}/${PDY}.${cyc}0000.cice_model.res.nc"
+  fi
+
```

Comment:
Minor change to match how the MOM restart is being done above (and remove a duplication)
```suggestion
  else
    sPDY=${PDY}
    scyc=${cyc}
  fi
  
  cice_restart_file="${COM_ICE_RESTART_PREV}/${sPDY}.${scyc}0000.cice_model.res.nc"
```